manifests/project.pp

```diff
@@ -5,7 +5,21 @@
 # to to build up the commands list
 #
 define puppi::project (
-  $enable = true ) {
+  $deploy_root              = undef,
```

Comment:
They are added in this PR. The problem is that the template for the project file is used from ```puppi::project```, so ```@deploy_root``` (f.ex.) is empty in the template. The solution is to add these as parameters.